### DISCUSSION TITLE
Subdomain only for postproccessing

### DISCUSSION THREAD

**Discussion by lyyc199586** [idaholab/moose#26633] [Q&A Meshing] [answered]
Can I create a subdomain only for postprocessing? For example, I want to create a box inside the main domain name `box`, but I don't want it to affect the current kernels (they will still running on the whole domain)

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can create a new block with any of the mesh generators, and add that new block to the block restriction of the kernels / the variables.

Guillaume

**Reply by lyyc199586** [reply]
ok, looks like that's the only way.

**Reply by GiudGiud** [reply]
you can use MeshDivisions or extra-element ids if you want too.

this does not modify the blocks so it wont influence the kernels.
There are a lot less postprocessors implemented for those techniaues though
